Cargo.toml

```diff
@@ -174,7 +174,7 @@ object = "0.36"
 once_cell = "1.18"
 opener = "0.7"
 parking_lot = "0.12"
-paste = "1.0"
+paste = "1.0.15"
```

Comment:
is there a reason we need to pin to this version? All of the crates in the sway repo _apart_ from fuel crates are using x.y instead of x.y.z

Reply:
No, no real reason, it was my overlook, I wasn't aware of the existing convention. I've changed it back to 1.0. I've also noticed that the `normalize-path` crate didn't follow the convention so I've changed that one too.